1.8/translations/translations.md

```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your translation to be default on forum so when users and guests access it you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
```

Comment:
If you want your translation to be default on forum so when users and guests access it you can change it by going to

Should be imo:

If you want your new translation to be default for all new users and guests you can change it by going to